docs/content/topics/operators/gateways.md

```diff
@@ -11,340 +11,518 @@ aliases:
 
 # Brigade Gateways
 
-This guide explains how gateways work, and provides guidance for creating your own
-gateway.
+This guide explains how gateways work, and provides guidance for creating your
+own gateway.
 
 ## What Is A Brigade Gateway?
 
-The [Brigade architecture](design.md) is oriented around the concept that Brigade
-scripts run as a response to one or more events. In Brigade, a _gateway_ is an
-entity that generates events. Often times, it translates some external trigger
-into a Brigade event.
+The [Brigade architecture](/topics/design) is oriented around the concept that
+Brigade scripts run as a response to one or more events. In Brigade, a gateway
+is an entity that generates events. Often times, it translates some external
+trigger into a Brigade event.
 
-Brigade ships with the ability to enable various gateways that are ready to go.
-
-These include the [Container Registry Gateway](dockerhub.md), the [Github Gateway](./github.md)
-and the [Generic Gateway](./genericgateway.md).  They can all be enabled via top-level
-Helm [chart flags](https://github.com/brigadecore/charts/blob/master/charts/brigade/values.yaml).
+While there is no default gateway that ships with Brigade, installing one
+alongside Brigade is as simple as a [Helm] chart install, along with any
+additional setup particular to a given gateway. See
+[below](#available-gateways) for a current listing of v2-compatible gateways.
 
 All of these provide HTTP-based listeners that receive incoming requests
-(from a container registry, Github or other platforms and systems) and generate
-Brigade events as a result.
+(from Github or other platforms and systems) and generate Brigade events as a
+result.
 
 However, Brigade's gateway system works with more than just webhooks.
 
-For example, the `brig` client also acts as a gateway. When you execute a `brig run`
-command, `brig` creates a Brigade event. By default, it emits an `exec` event. And
-Brigade itself processes this event no differently than it processes the GitHub
-or container registry events.
+For example, the `brig` client also acts as a gateway. When you execute a
```

Comment:
Willie and I had some back and forth over this with noisy neighbor-- which I elected not to count as a "gateway."

Unless I'm mistaken, in general (not Brigade, specifically) "gateways" are generally considered to be components that bridge / join / mediate between two systems...

I'm not sure if the CLI qualifies.

If you agree, this could perhaps be rectified by saying "the `brig` client behaves similarly to a gateway."